File: donghyo/Programmers/notWanzoo.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;
using namespace __gnu_cxx;

string solution(vector<string> participant, vector<string> completion) {
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    
    for(int i=0; i<participant.size(); i++)
        if(participant[i] != completion[i])
            return participant[i];
    return participant[participant.size()-1];
    
}

File: donghyo/Programmers/notWanzoo.py
def solution(participant, completion):
    participant.sort()
    completion.sort()

    for i, j in zip(participant, completion):
        if i != j:
            return i

    return participant[-1]


participant = ["marina", "josipa", "nikola", "vinko", "filipa"]
completion = ["josipa", "filipa", "marina", "nikola"]
solution(participant, completion)